cmd/image/import/import.go

```diff
@@ -106,7 +107,13 @@ pvsadm image import -n upstream-core-lon04 -b <BUCKETNAME> --object-name rhel-83
 			}
 			klog.Infof("%s bucket found in the %s[ID:%s] COS instance", opt.BucketName, *cosOfBucket.Name, *cosOfBucket.ID)
 
-			// Step 2: Create the Service Credential in the found COS instance
+			//Step 2: Check if object Exists or not
+			objectExists := s3client.CheckIfObjectExists(opt.BucketName, opt.ImageFilename)
```

Comment:
This flow does not cover the if the user feeds the access key and secret key,

Reply:
checking s3 object existence is independent of access key, secret key... s3 api's are independent of the access key/secretkey

Reply:
Yes.. I missed the if block